src/app/components/listbox/listbox.css

```diff
@@ -19,6 +19,7 @@
     cursor: pointer;
     font-weight: normal;
     margin-bottom: 1px;
+    position: relative;
```

Comment:
this is to enable the scroll to move down and up when press arrows keys.
that is because `ui-helper-hidden-accessible` get `position: absolute`
same thing I see in `ui-multiselect-item`